### PR TITLE
Added linters and suggested changes made

### PR DESCRIPTION
**Pull request by rbreva**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<!DOCTYPE html>
 <html lang="en">
 
 <head>
```
Comment (rbreva) [inline index.html:4]: Please you should use a more descriptive title,

File: index.html [hunk 2]
```diff
@@ -48,8 +49,8 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
                 <label for="not-sure">Not sure</label>
             </div>
 
-            <label id="favorite-feature-label" for="favorite-featur">What is your favorite feature of freeCodeCamp?</label>
-            <select name="favorite-feature" id="drop-down">
+            <label id="favorite-feature-label" for="favorite-feature">What is your favorite feature of freeCodeCamp?</label>
+            <select name="favorite-feature" id="favorite-feature">
                 <option value="0" selected disabled>Select an option</option> 
                 <option value="1">Challenge</option>
                 <option value="2">Projects</option>
```
Comment (rbreva) [inline index.html:56]: I think it is necessary to review the use of classes and ids

**Review by rbreva — COMMENTED** [review]
# Changes required ♻️
 
Please add readme.md

Submit for a re-review when you're done. 👨🏼‍💻 

Thanks 👍🏼

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @rbreva, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- Good use of GitHub features 👍 

## Required Changes ♻️
- [ ] **Great job using inline comments and detecting issues in the code 👏 . Kindly make sure you explain why a review was given to a student so they can understand it better 🙏**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rbreva — COMMENTED** [review]
### Changes required ♻️

Kindly, It is necessary to take into account that the same id should not be used more than once on the same page. 👍🏼

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @rbreva ✋



Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Optional suggestions

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**